checkov/secrets/runner.py

```diff
@@ -128,6 +128,22 @@ def run(
                     'limit': ENTROPY_KEYWORD_LIMIT
                 }
             ]
+        custom_plugins = os.getenv("CHECKOV_CUSTOM_DETECTOR_PLUGINS_PATH", None)
```

Comment:
```suggestion
        custom_plugins = os.getenv("CHECKOV_CUSTOM_DETECTOR_PLUGINS_PATH")
```
`None` is always the default, so no need to extra add it here 😄